nuget/lib/dependabot/nuget/native_helpers.rb

```diff
@@ -55,17 +55,11 @@ def self.run_nuget_framework_check(project_tfms, package_tfms)
         false
       end
 
-      # rubocop:disable Metrics/MethodLength
       sig do
-        params(
-          repo_root: String,
-          proj_path: String,
-          dependency: Dependency,
-          is_transitive: T::Boolean,
-          credentials: T::Array[T.untyped]
-        ).void
+        params(repo_root: String, proj_path: String, dependency: Dependency,
+               is_transitive: T::Boolean).returns(T::Array[String])
```

Comment:
Should this be the ruby equivilant of a tuple (whatever that is) instead of an array since it will only ever have 2 elements in defined positions?

Reply:
Tuples are still experimental in Sorbet, but you can definitely try them: https://sorbet.org/docs/tuples

Reply:
Ruby doesn't have tuples; it's recommended (at least in the documentation I found) to do it this way.

Reply:
@JamieMagee I didn't know this; just added and got rid of the `T.must(...)`